src/orbit.jl

```diff
@@ -6,7 +6,8 @@
 struct Orbit     
   P::Float64             # days             # QUERY:  Should we store P or a here?
   ecc::Float64
-  incl::Float64          # radians
+  incl_mut::Float64      # radians; mutual inclination relative to a reference (e.g. invariant) plane
```

Comment:
What is the reason to include both of these in the Orbit struct?   
I'm nervous that adding both of these means we need to keep everthing self-consistent.  
Could the same thing be accomplished by storing the inclination and ascending node of the reference plane in the PlanetarySystem struct and then having a function to compute the inclination of the orbit relative to the reference plane?

Reply:
I want to save the inclinations of the planets relative to their reference plane.
Yes, we could accomplish the same by storing the inclination of the reference plane and the ascending node of each planet's orbit (already in the Orbit struct), and then recomputing the mutual inclinations afterwards. I see that the PlanetarySystem struct has three fields: star, planet, and orbit; we could add a fourth for the inclination of the system/reference plane but wouldn't this cause the same issue, i.e. force us to revise all the other code that uses this struct?